C/c4Certificate.cc

```diff
@@ -552,7 +552,8 @@ namespace c4Internal {
         }
 
         virtual fleece::alloc_slice publicKeyRawData() override {
-            return publicKeyDERData();
+            Retained<PublicKey> publicKey = new PublicKey(publicKeyDERData());
```

Comment:
Separate these into two lines please.  Windows often has issues with inline `alloc_slice` passed to a `slice` argument because it will get deallocated before it is used.

Reply:
Done